src/platform/common/utils/localize.ts

```diff
@@ -214,25 +238,28 @@ export namespace DataScience {
 
     export const dataExplorerTitle = localize('DataScience.dataExplorerTitle', 'Data Viewer');
     export const badWebPanelFormatString = localize(
-        'DataScience.badWebPanelFormatString',
+        {
+            key: 'DataScience.badWebPanelFormatString',
+            comment: ['{RegEx=">[^<]*<"}', 'Only translate the text within the HTML tags']
+        },
         '<html><body><h1>{0} is not a valid file name</h1></body></html>'
     );
     export const checkingIfImportIsSupported = localize(
-        'DataScience.checkingIfImportIsSupported',
-        'Checking if import is supported'
+        { key: 'DataScience.checkingIfImportIsSupported', comment: ['{Locked="import"}'] },
+        'Checking if "import" is supported'
```

Comment:
I am assuming we want to keep `import` as is and not change it from language to language.